packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayout.types.ts

```diff
@@ -141,6 +139,15 @@ export interface IDashboardGridSectionLayoutProps extends IDashboardGridLayoutPr
    * @default false
    */
   isCollapsible?: boolean;
+  /**
+   * Callback so you can save the layout.
+   */
```

Comment:
NIT: extra line

Reply:
resolved

---
In reply to: [216082115](https://github.com/OfficeDev/office-ui-fabric-react/pull/6285#discussion_r216082115) [](ancestors = 216082115)